swerve_drive_controller/package.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>swerve_drive_controller</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="nitinmaurya2606@gmail.com">knight</maintainer>
+  <license>Apache-2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
+  <!-- <build_depend>generate_parameter_library</build_depend> -->
```

Comment:
why uncommented? should be of `<depend>` level
https://github.com/PickNikRobotics/generate_parameter_library?tab=readme-ov-file#add-parameter-library-generation-to-project